templates/single.twig

```diff
@@ -35,25 +35,11 @@
 									data-ga-label="n/a">
 										{{ issue.name|e('wp_kses_post')|raw }}
 								</a>
-							{% endfor %}
-						</div>
-					{% endif %}
 
-					{% if ( post.issues_nav_data and post.tags ) %}
-						<span class="tag-wrap-bullet" aria-hidden="true">&#8226;</span>
-					{% endif %}
+								{% if ( loop.last == false ) %}
+									<span class="category-separator" aria-hidden="true">|</span>
```

Comment:
Have you tried to add it by the `content` property directly from the css? 
Like this:
![Screenshot 2022-01-26 at 09 05 41](https://user-images.githubusercontent.com/77975803/151160158-ab95df6f-aba8-4e7b-b208-5f6cf95b7574.png)

This makes to the selector to be more reusable and avoid to forget to add the pipe in some other case

Reply:
These separators actually [used to be implemented as pseudo-elements](https://github.com/greenpeace/planet4-master-theme/commit/6fa1fc1e44ff3d1f0560bb27b56908c6b13a4e02), but I think that was changed for accessibility purposes 🤔